pkg/packer/manifests/azure/packer.pkr.hcl

```diff
@@ -355,6 +355,7 @@ source "azure-arm" "kib_image" {
   managed_image_name                = local.managed_image_name
   managed_image_resource_group_name = var.resource_group_name
   os_type                           = "Linux"
+  os_disk_size_gb                   = 80
```

Comment:
lets make this a packer variable with default to `80`

Reply:
for sure, much cleaner. just wanted to capture it here before forgetting